modules/twitch_streaming

```diff
@@ -0,0 +1,71 @@
+"""
+Checks if a streamer is online. [Gross Gore/grossie_gore as placeholder]
+
+Checks if a streamer is online using the Twitch Kraken API to see
+if a channel is currently streaming or not.
+
+Configuration parameters
+    cache_timeout: how often we refresh this module in seconds
+    stream_name: name of streamer(twitch.tv/<stream_name>)
+
+Format of status string placeholders
+    {stream_name}:  name of the streamer (twitch.tv/<stream_name>)
```

Comment:
being picky `{stream_name}` can be the stream name or the display name so I'd just remove the '(twitch.tv/<stream_name>)' bit here